### PR TITLE
Nano aod angel

### PR DESCRIPTION
**Pull request by arosado1**
Updating READ.md by adding "mkdir plots" and changing ZJetsToNuNu_2016

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ cd $CMSSW_BASE/src
 git clone git@github.com:susy2015/ZInvisible.git
 cd ZInvisible/Tools
 ```
+mkdir plots
```
Comment (caleb-james-smith) [inline README.md]: can you put this within the code section (before ```)

Comment (caleb-james-smith) [reply README.md]: that way it will be formatted the same way; we use this format for terminal commands

Comment (arosado1) [reply README.md]: I move it already

Comment (caleb-james-smith) [reply README.md]: I see it now. Thanks!

**Comment by caleb-james-smith** [comment]
Looks good.

**Pull request merged**